### ISSUE TITLE
[Feature Request]: `KryptonTaskDialog` Features

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#1339]
Ensure that the current `KryptonTaskDialog` has a comparable feature set to the native WinForms implementation, found in .NET 5 and later.

https://learn.microsoft.com/en-gb/dotnet/api/system.windows.forms.taskdialog?view=windowsdesktop-8.0

**Comment by Wagnerp** [comment]
PR https://github.com/dotnet/winforms/pull/1133

**Comment by Smurf-IV** [comment]
Current version has the following functionality:
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/f2551502-abdb-4df5-9ce3-8b58914e9ac6)

So Can you confirm the checklist below please ? (https://github.com/dotnet/winforms/issues/146#issue-387449844)
- [ ]  Add a way to add Custom `TaskBarButton`'s that have user specified text
- [ ] `Main Instructions` to be replaced with a `Heading`
- [ ] `Heading area to be coloured related to icon in use i.e. `TaskDialogIcon.ShieldWarningYellowBar`
- [ ] Have programmatic expandable area
- [ ] Richer usage of the `CommandLink Buttons` (see Custom `TaskBarButton`)
- [ ] Progress bar to be placed under "Content" area
- [ ] Make `KryptonTaskDialogIcon` (etc.) compatible with ALL version of supported .net Framework
- [ ] Ensure that this can be used modeless
- [ ] Ensure Designer is easy to use (https://github.com/dotnet/winforms/issues/146#issuecomment-451130343)
- [ ] If not possible to make it "Backward compatible" then have this as a new Class ?

**Comment by Wagnerp** [comment]
@Smurf-IV Yes that's fine. I think that the current implementation still uses a design philosophy from 2006 - 2007. TDs have moved on since then.

**Comment by Wagnerp** [comment]
Is it also possible to include the `banner` animations to task dialogs?

**Comment by Smurf-IV** [comment]
> Is it also possible to include the `banner` animations to task dialogs?

Can you modify the original image with an animated gif showing that please, and also add it as one of the checkbox items

**Comment by Wagnerp** [comment]
> > Is it also possible to include the `banner` animations to task dialogs?
> 
> Can you modify the original image with an animated gif showing that please, and also add it as one of the checkbox items

[Here](https://blog.prototypr.io/windows-copy-function-followed-form-11cf4bf6a87e) (Windows Vista/7). I think the AVI's are located in `shell32`.

**Comment by Wagnerp** [comment]
![1339](https://github.com/Krypton-Suite/Standard-Toolkit/assets/949607/e0aef284-692b-4f1e-992f-979ef6e78b8d)

**Comment by Smurf-IV** [comment]
In that case: No.
Why: 
- Because it means there has to be an animation sequence for each of the Base themes to ensure that it "Fits in".
- Also. copying is not really a Winform Control.
- And Animations should be in the Ext toolkit, and once there is is at least a light / normal (As above) / Dark animation then it dow not make sense

**Comment by Wagnerp** [comment]
@Smurf-IV Going to make the checklist into separate feature requests, similar to what I've done with toasts if this is ok with you?

**Comment by Smurf-IV** [comment]
Makes sense,
Then their related #'s can be tallied back to this checklist.

**Comment by giduac** [comment]
Continued in #1376, closing
